python/ribasim/ribasim/validation.py

```diff
@@ -0,0 +1,46 @@
+# Table for connectivity
+# "basin": ["linear_resistance"] means that the downstream of basin can be linear_resistance only
+connectivity = {
```

Comment:
Maybe give it a type hint?

Reply:
Well that one is deprecated. I meant `connectivity: dict[str, list[str]] = {...`